cg/store/models.py

```diff
@@ -6,13 +6,18 @@
 from sqlalchemy import Column, ForeignKey, orm, types, UniqueConstraint, Table
 
 from cg.constants import (
-    REV_PRIORITY_MAP,
-    PRIORITY_MAP,
     FAMILY_ACTIONS,
     FLOWCELL_STATUS,
+    Pipeline,
     PREP_CATEGORIES,
+    PRIORITY_MAP,
+    REV_PRIORITY_MAP,
+    SEX_OPTIONS,
+    STATUS_OPTIONS,
 )
 
+PIPELINE_OPTIONS = [pipeline.value for pipeline in Pipeline]
```

Comment:
Suggestion: create members method to get all the members instead of making for loop

Reply:
this is not part of the code anymore, didn't implement members explicitly though